code/README.md

```diff
@@ -26,6 +26,15 @@ Setup requirements
  - python3, libpython3-dev
  - ctypes, matplotlib, numpy python packages
 
+### A word on timing
+
+  - several exercises ask you to "time" things
+  - here it's always sufficient to use the `time` command line:
+  ```shell
+  time ./racing
+  ./racing  0.02s user 0.03s system 135% cpu 0.031 total
```

Comment:
The output looks a lot different on my Ubuntu:
```
bgruber@graviola:~/dev/cpluspluscourse/code/race$ time ./racing.sol 

real	0m0.022s
user	0m0.011s
sys	0m0.021s
```
I don't have a `total`. I should use `real` then?

Reply:
ok, moved to POSIX output (I need a -p/--portability to have this format) and took the real time. Any time is fine for comparisons anyway.